package.json

```diff
@@ -40,7 +40,8 @@
     "prepublishOnly": "npm run build",
     "test": "jest --coverage",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "storyshot": "jest --coverage --coverageDirectory=coverage/visual-regression"
```

Comment:
Are we planning on having more than one storyshot test, or is it one storyshot test that tests all components? If it's the latter, would it be a good idea to shorten the command to maybe npm run storyshot?